src/routes/app/(app)/projects/(forges)/github/[githubUsername]/[githubRepoName]/+page.server.ts

```diff
@@ -11,17 +10,18 @@ import type { ProjectByUrlQuery } from './__generated__/gql.generated';
 import type { QueryProjectByUrlArgs } from '$lib/graphql/__generated__/base-types';
 import isClaimed from '$lib/utils/project/is-claimed';
 import { PROJECT_PROFILE_FRAGMENT } from '../../../components/project-profile/project-profile.svelte';
+import { Octokit } from '@octokit/rest';
+import { GITHUB_PERSONAL_ACCESS_TOKEN } from '$env/static/private';
+import GitHub from '$lib/utils/github/github';
 
-// TODO: This fails if the network is not the default one. We need to support other networks.
+const octokit = new Octokit({ auth: GITHUB_PERSONAL_ACCESS_TOKEN });
+const github = new GitHub(octokit);
 
 export const load = (async ({ params, fetch }) => {
   const { githubUsername, githubRepoName } = uriDecodeParams(params);
 
-  const gitHubUrl = `https://github.com/${githubUsername}/${githubRepoName}`;
-
-  if (!(await siteExists(gitHubUrl))) {
```

Comment:
What would happen now if the site does not exist?

Reply:
not the right thing — fixed!